### ISSUE TITLE
Not Getting Anything out of C1 GPIO

### ISSUE THREAD

**Issue by GeneralReparm** [H4ckd4ddy/flipperzero-sentry-safe-plugin#5]
Greetings. I've compiled this app under the current firmware (0.74.2) using the instructions you provided. I successfully complied and loaded the app onto my flipper (which is running official firmware 0.74.2). The app launches successfully, but when I analyze the serial output after pressing "okay" in your app with something like Putty, I see no signal. Alternatively, when I use your Arduino sketch from your other project and monitor the serial output, I do see data appear. Do I need to be on unleashed firmware in order for this app to work correctly? Why am I not seeing any output from the flipper, but I do when I test it on an Arduino?

**Comment by sdgros01** [comment]
im at a loss as well. Mine will install run and said done but nothing ever happens at all

**Comment by H4ckd4ddy** [comment]
Hi,
First sorry for the delay.
Unleashed firmware is not required for this plugin. If you can run the plugin without error you should have some output on C1. Maybe can you just plug an LED between GND and C1 to see if it blink. If it slightly blink, all is fine. If not, the here maybe an hardware issue on your flipper :(
Let me know if you managed to use it.
Best regards

**Comment by GeneralReparm** [comment]
> Hi,
> First sorry for the delay.
> Unleashed firmware is not required for this plugin. If you can run the plugin without error you should have some output on C1. Maybe can you just plug an LED between GND and C1 to see if it blink. If it slightly blink, all is fine. If not, the here maybe an hardware issue on your flipper :(
> Let me know if you managed to use it.
> Best regards

No problem at all. I ended up delving into this deeper and purchased a logic analyzer much like the one on the Arduino project board for sentry safe. Using that logic analyzer I was able to confirm that the correct output is coming out of C1 on the flipper. I got my safe to reset the code to 0000 ONCE. This was switching between my Arduino and my flipper and by the time I realized that it worked, I had lost track unfortunately so unclear which did it but…it definitely did it somehow. Anyways, I can confirm the output is correct from the flipper now. It must be something about my connection to the safe panel itself (the connection to flipper / logic analyzer worked perfectly). Not sure what’s going on but I’ve never been able to reset the safe code to 0000 again. The flipper program does not appear to be the culprit though.
